docs/output.md

```diff
@@ -22,32 +23,46 @@ The pipeline is built using [Nextflow](https://www.nextflow.io/) and processes d
 [Guppy](https://nanoporetech.com/nanopore-sequencing-data-analysis)
 
 *Description*:  
-Guppy will demultiplex and barcode the data given from an ONT device. The flowcell, kit and barcode kit must be given in the command line if demultiplexing needed. This step can by bypassed using the `--skip_demultiplexing` parameter when initiating the pipeline. The output folders will be separated into the barcodes from the kit used and unclassified.
+Guppy will demultiplex and barcode the data given from an ONT device. The flowcell, kit and barcode kit must be given in the command line if demultiplexing needed. This step can by bypassed using the `--skip_demultiplexing` parameter when initiating the pipeline. The output folders will be separated into the barcodes from the kit used and unclassified. The output in each barcode folder is then merged into one fastq file for easier downstream processesing. 
 
 *Output directories*:
 
-* `guppy/barcode*/`  
+* `guppy/basecalling/barcode*/`  
   FastQ files output for each barcode
-* `guppy/unclassified/`  
+* `guppy/basecalling/unclassified/`  
   FastQ files output that are unclassified
-* `guppy/sequencing_summary.txt`
+* `guppy/fastq`
+  Merged output of fastq files into one fastq for each barcode
 
-## Quality Control
+## Sequncing Quality Control
 
 *Documentation*:  
 [PycoQC](https://github.com/a-slide/pycoQC), [NanoPlot](https://github.com/wdecoster/NanoPlot)
 
 *Description*:  
-PycoQC and NanoPlot give general quality metrics about your reads. It provides information about the quality score distribution across your reads, read lengths and other general stats.
+PycoQC and NanoPlot give general quality metrics about the sequencing run. It provides information about the distribution of read length, read length over time, number of reads per barcode and other general stats.
+![PycoQC - Number of Reads per Barcode plot](images/NumberofReadsperBarcode.png)
 
 *Output directories*:
 
 * `pycoQC/`  
   An .html file output is produced that includes a run summary and graphical representation of distribution of read length, distribution of read quality scores, mean read quality per sequence length, output per channel over experiment time, output over experiment time, read quality over experiment time, readlength over experiment time, and percentage of reads per barcode.
 * `nanoplot/summary/`  
-  An output of a .png file of a statistical summary and an html summary file.
+  An output of a .png file of a statistical run summary and an html summary file of overall run.
```

Comment:
What do you mean by `statistical run summary` and `overall run`?

Reply:
Just to differentiate it between the individual fastqc Nanoplot report from the overall Nanoplot report. I will reword this to be more clear.

Reply:
That would be perfect, because I honestly had no idea what you were talking about.
:confused: